src/runtime/lpython/lpython.py

```diff
@@ -859,3 +707,11 @@ def __call__(self, *args, **kwargs):
         function = getattr(__import__("lpython_module_" + self.fn_name),
             self.fn_name)
         return function(*args, **kwargs)
+
+def bitnot(x, bitsize):
+    return (~x) % (2 ** bitsize)
+
+bitnot_u8 = lambda x: bitnot(x, 8)
+bitnot_u16 = lambda x: bitnot(x, 16)
+bitnot_u32 = lambda x: bitnot(x, 32)
+bitnot_u64 = lambda x: bitnot(x, 64)
```

Comment:
Why not just calling it bitnot? And make the function work for all u8 - u64.

Reply:
You are right, I didn't realize that. CPython puts quite a lot of limitations in what we can do, but I think it is worthy to just use the built-in `int` and `float` and build everything on top.

Let's use what you did then.